DEVELOPING.md

```diff
@@ -24,6 +24,42 @@ To do a full clean run:
 yarn clean_all
 ```
 
+## Debugging
```

Comment:
I don't think this is the right file - users aren't directed to look at anything outside docs/

Reply:
👍 moved to /docs/install.md